ivy.el

```diff
@@ -196,6 +196,12 @@ See https://github.com/abo-abo/swiper/wiki/ivy-display-function."
           (const :tag "Popup" ivy-display-function-popup)
           (const :tag "Overlay" ivy-display-function-overlay)))
 
+(defvar ivy-display-functions-props
+  '((ivy-display-function-overlay :cleanup ivy-overlay-cleanup))
+  "Map display functions of ivy to theirs action property lists.
+These actions are functions which work closely with display functions,
+for example: cleanup action functions.")
```

Comment:
I think it's better to write:

```el
 "Map Ivy display functions to their property lists.
Examples of properties include associated `:cleanup' functions."
```

There's no reason to limit this alist to "action" functions.